cmd/soci/commands/debug.go

```diff
@@ -0,0 +1,38 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"net/http"
+	"oras.land/oras-go/v2/registry/remote"
+)
+
+type debugClient struct {
```

Comment:
Can this be part of `push.go`? The current convention is that each file/folder under `commands/` represents a top-level command.

Reply:
Sure